task_2.js

```diff
@@ -1,7 +1,7 @@
 function compare(arr1, arr2) {
-    if (arr1.length != arr2.length) return false
-    else {
-        for (var i = 0; i < arr1.length; i++)
-            if (arr1[i] === arr2[i]) return true
-    }
+    if (arr1.length === arr2.length) {
+        for (var value of arr1)
+            if (arr1[value] === arr2[value]) return true
+            else return false
```

Comment:
get away 'else' constructnions

Reply:
Done

Reply:
There is a more simple way to return without if-else . Try it please